db/migrate/20230515151440_change_katalyst_content_items_show_heading_column.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class ChangeKatalystContentItemsShowHeadingColumn < ActiveRecord::Migration[7.0]
+  def up
+    add_column :katalyst_content_items, :heading_style, :string, null: false
```

Comment:
Why string rather than integer? You can do string enums in rails but it doesn't seem that useful in this case (to me).

Why don't you set a default value in the migration? E.g. `default: "none"` (or `default: 0` for integer enum)